src/components/Amount/Amount.re

```diff
@@ -0,0 +1,52 @@
+open Transactions;
+
+[@react.component]
+let make = (~transactions: t) => {
+  let rupee = React.string({js|₹|js});
+
+  let expenseIncomeStyle =
+    ReactDOM.Style.make(
+      ~backgroundColor="white",
+      ~display="flex",
+      ~justifyContent="space-between",
+      ~boxShadow="-2px 3px 3px 0px rgba(168, 168, 168, 0.75)",
+      ~borderRadius="4px",
+      (),
+    );
+  let incomeStyle = ReactDOM.Style.make(~flex="1", ~textAlign="center", ());
+  let incomeH5Style =
+    ReactDOM.Style.make(~color="green", ~fontSize="20px", ());
+  let expenseStyle = ReactDOM.Style.make(~flex="1", ~textAlign="center", ());
+  let expenseH5Style =
+    ReactDOM.Style.make(~color="red", ~fontSize="20px", ());
+
+  let amountArray = Belt.Array.map(transactions, t => t.amount);
+  let incomeArray = Belt.Array.keep(amountArray, i => i > 0);
+  let expenseArray = Belt.Array.keep(amountArray, i => i < 0);
```

Comment:
As an exercise, try to rewrite this using [this concept](https://reasonml.github.io/docs/en/function#partial-application)\

While you are at it, also read about [currying vs partial application](https://betterprogramming.pub/functional-programming-currying-vs-partial-application-53b8b05c73e3)

same but different but same but different